packages/vite-plugin-cloudflare/e2e/remote-bindings.test.ts

```diff
@@ -1,7 +1,8 @@
 import assert from "node:assert";
+import { randomUUID } from "node:crypto";
```

Comment:
I think the additions here are incorrect/unintentional, these imports are indeed not being used anyways:
<img width="672" height="169" alt="Screenshot 2025-08-21 at 22 09 24" src="https://github.com/user-attachments/assets/eb79a696-594d-44c9-b8c1-110c68d0c1e7" />